exception.test=[Exception] 테스트